### ISSUE TITLE
v2.01 null modem cable is broken

### ISSUE THREAD

**Issue by bluecursor** [dirkwhoffmann/vAmiga#697]
No longer links up with TCPSER. Was working when option was first released (vAmiga.1.1b9)

**Comment by dirkwhoffmann** [comment]
Confirmed. Sockets are no longer working.

The code throws an `ERROR_SOCK_CANT_BIND` exception in this function:

```
void
Socket::bind(u16 port)
{
    // Create the socket if it is uninitialized
    create();

    struct sockaddr_in address;
    address.sin_family = AF_INET;
    address.sin_addr.s_addr = INADDR_ANY;
    address.sin_port = util::bigEndian(port);
    
    if (::bind(socket, (struct sockaddr *)&address, sizeof(address)) < 0) {
        throw VAError(ERROR_SOCK_CANT_BIND);
    }
}
```

**Comment by dirkwhoffmann** [comment]
Just another sandboxing issue... vAmiga was missing the following two entitlements:

    com_apple_security_network_server
    com_apple_security_network_client

**Comment by dirkwhoffmann** [comment]
Fixed in v2.1b2
